packages/pancake-profile-sdk/README.md

```diff
@@ -0,0 +1,185 @@
+# Pancakeswap Profile SDK
+
+This package provides some handy functions to retrieve data for Pancakeswap Profile system.
+
+If you're looking for React-ready solution - take a look at the [profile-hook](https://github.com/pancakeswap/pancake-toolkit/tree/master/packages/pancake-profile-hook).
+
+##### Table of Contents
+
+- [Installation](#installation)
+- [Usage](#usage)
+  - [getUsername](#getUsername)
+  - [getTeam](#getTeam)
+  - [getProfile](#getProfile)
+  - [getAchievements](#getAchievements)
+- [Roadmap](#roadmap)
+
+## Installation
+
+Install `@pancakeswap-libs/profile-sdk` into your project with npm:
+
+```bash
+npm install @pancakeswap-libs/profile-sdk --save
+```
+
+or yarn:
+
+```bash
+yarn add @pancakeswap-libs/profile-sdk
+```
+
+This package requires `web3` to be installed in your project. If you're using TypeScript you also should instal `web3-eth-contract` and `web3-utils` to avoid type errors, although depending on your TypeScript and ESlint configuration you might skip installing those (since they are sub-packages of web3).
```

Comment:
A note for the future - we should consider supporting `ether.js`. Maybe let the consumer provide the library.